test/blackbox-tests/test-cases/enabled_if/eif-public-name-collision.t

```diff
@@ -0,0 +1,64 @@
+Public libraries using the same `public_name`, in different contexts
```

Comment:
Could you describe the test a bit more here? It should be possible to know what it's about without knowing any context.